tests/routine/test_parse_routine.py

```diff
@@ -52,37 +53,6 @@ def test_raw_routine_from_file(self):
         )
         assert result.dependencies == ["udf.test_bitmask_lowest_28"]
 
-    def test_raw_routine_from_text(self):
```

Comment:
Yeah, I removed the `from_text` parameter from `RawRoutine.from_file` since it wasn't used anywhere in the code base except for this test, which is just checking that the `from_text` parameter works correctly.